mobile/api.go

```diff
@@ -2,19 +2,63 @@ package mobile
 
 import (
 	"encoding/json"
+	"encoding/hex"
 
+	"github.com/skycoin/skycoin/src/cipher"
 	"github.com/skycoin/skycoin-lite/liteclient"
+	"github.com/skycoin/skycoin/src/cipher/go-bip39"
 )
 
 // GetAddresses returns a series of addresses based on a seed and the number of addresses
-// Seed is be hex-encoded bytes.
-func GetAddresses(seed string, amount int) (string, error) {
-	addresses := liteclient.GenerateAddresses(seed, amount)
+// Seed should be the mnemonic word list
+func GetAddresses(seed string, num int) (string, error) {
```

Comment:
Most of the code of this function could be deleted simply by calling `liteclient.GenerateAddresses`, which would have the advantage of avoiding repeating the code and that there are already some tests for `liteclient.GenerateAddresses`. The web wallet uses that function and works correctly with the seeds made with the desktop wallet. Is there any particular reason for repeating the code here?

Another thing is that, as returning multiple values is something not very common in other languages, until now almost all the functions on this repository call `panic()` in case of error instead of returning the error, to throw exceptions (this function maintained the Go format because it is old code that had not been touched and was not in use). In fact, in the `PrepareTransaction` function below it would work like this, because `liteclient.PrepareTransaction` calls `panic`. If there is no particular reason for returning the error, it might be good to maintain the standard used so far.

Reply:
I remember there was *some* reason I did the GenerateAddresses again but I cant remember why :-P I'll take a look at it and change it or find the reason.

I think I found that the gomobile java-generator automatically wraps the "val,error" format into throwing an exception if the error is not nil, but I dont really care about it so I'll change it to the panic() way of doing it if you prefer it. I'll update this PR soon.